coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -2,23 +2,28 @@
 
 
 @generate_repr()
-@generate_eq("documentation", "marker", "range")
+@generate_eq("documentation", "language", "docstyle", "marker", "range")
 class DocumentationComment:
     """
     The DocumentationComment holds information about a documentation comment
     inside source-code, like position etc.
     """
 
-    def __init__(self, documentation, marker, range):
+    def __init__(self, documentation, language,
+                 docstyle, marker, range):
         """
         Instantiates a new DocumentationComment.
 
         :param documentation: The documentation text.
-        :param marker:        The three-element tuple with marker strings that
-                              identified this documentation comment.
+        :param language:      The language of the documention.
+        :param docstyle:      The docstyle used in the documentation.
+        :param marker:        The three-element tuple with marker strings,
+                              that identified this documentation comment.
         :param range:         The position range of type TextRange.
         """
         self.documentation = documentation
+        self.language = language
+        self.docstyle = docstyle
```

Comment:
I think we should do that :+1:, so there's no ambiguity e.g. for `Python`, `python` or similar :)
But mention this inside docs :+1: